kylecornelissen/whats-cookin
# Filter tags

#### What's this PR do?
- Creates a function to find all unique tags within recipe data
- Creates a function to determine which checkboxes are checked when filter button is clicked
- Creates a function to find all recipes associated with any checked box's tag
- Creates a function to hide any recipes that do not have the checked box's tag from the DOM
- Fixes linter errors
- Removes placeholder code from HTML and CSS
- Extends sidebar to bottom of page even after recipes are filtered
#### Where should the reviewer start?
Open up webpage with the updated code
#### How should this be manually tested?
Check one or more boxes on the sidebar and click filter. Try scrolling down after you have filtered out recipes.
#### Any background context you want to provide?
N/A
#### What are the relevant tickets?
#25
